project_national/national_baseline.yml

```diff
@@ -32,7 +32,7 @@ workflow_generator:
       - scenario_name: Bills
 
     simulation_output_report:
-      timeseries_frequency: hourly
+      timeseries_frequency: 15min
```

Comment:
You'd want "timestep" here. And then you'd change `simulation_control_timestep` above to "15".